doc/source/text.rst

```diff
@@ -429,6 +429,27 @@ String ``Index`` also supports ``get_dummies`` which returns a ``MultiIndex``.
 
 See also :func:`~pandas.get_dummies`.
 
+Concatenating Series
+--------------------
+
+The method :meth:`Series.str.cat` can be used to concatenate the records of two :class:`Series`. Depending on the value given to the ``align`` keyword,
```

Comment:
can you describe the simpler usecase first (IOW just concat with no other, or other is a simple list). These can just be examples, doesn't have to be so much text.

Reply:
In my defense, so far there was no description about ``str.cat`` in ``text.rst``. But I will try to write up an overview. Where should it be placed, in your opinion? I would say directly after the splitting-section (natural opposites).